app/javascript/src/components/Dashboard/Notes/DeleteAlert.jsx

```diff
@@ -31,9 +21,7 @@ const DeleteAlert = ({
       onSubmit={handleDelete}
       onClose={onClose}
       message="Are you sure you want to continue? This cannot be undone."
-      title={`Delete ${selectedNoteIds.length} ${
-        selectedNoteIds.length > 1 ? "notes" : "note"
-      }?`}
+      title={"Delete note?"}
```

Comment:
No need for the braces here. [Refer](https://academy.bigbinary.com/learn-react/pages/miscellaneous-tips#:~:text=Rendering%20strings%20and%20blank%20spaces%20in%20JSX).